### PR TITLE
[Serve] [Doc] Add section on integrating with model registries

### PR DESCRIPTION
**Pull request by architkulkarni**
## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [x] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: doc/source/serve/ml-models.rst [hunk 1]
```diff
@@ -64,6 +64,38 @@ That's it. Let's take a look at an example:
...
+.. code-block:: python
+
+  import mlflow.pyfunc
+
+  class MLflowBackend:
+      def __init__(self, model_uri):
+          self.model = mlflow.pyfunc.load_model(model_uri=model_uri)
+
+      async def __call__(self, request):
+          df = await request.body() # pandas DataFrame
```
Comment (simon-mo) [inline doc/source/serve/ml-models.rst:87]: how do you get this into a pandas dataframe? it's not very obvious here.
How about:
```suggestion
          csv_text = await request.body() # The body contains just raw csv text.
          df = pd.read_csv(csv_text)
```

**Review by simon-mo — APPROVED** [review]

File: doc/source/serve/ml-models.rst [hunk 2]
```diff
@@ -64,6 +64,38 @@ That's it. Let's take a look at an example:
...
+          return self.model.predict(df)
+
+A Serve backend can then be created to serve the model as follows:
+
+.. code-block:: python
+
+  client.create_backend("my_backend", MLflowBackend, my_model_uri)
+
+For an even more hands-off and seamless integration with MLflow, check out the Ray Serve
+MLflow deployment plugin at https://github.com/ray-project/mlflow-ray-serve.  A full
```
Comment (simon-mo) [inline doc/source/serve/ml-models.rst:97]: can you make these rst links?

Comment (architkulkarni) [reply doc/source/serve/ml-models.rst:98]: Good catch, thanks

**Review by edoakes — APPROVED** [review]
Add a small note that this isn't limited to MLflow, it could be used with any model registry

**Pull request merged**